api/cart.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const cartRouter = express.Router;
+const { Cart } = require('../db/models/index');
+
+cartRouter.post("/", async (req, res, next) => {
+    try {
+        const { orderId, quantity } = req.body;
+        const newOrder = await addItemToCart ({ orderId, quantity });
```

Comment:
- Now that you have `{ Cart }` deconstructed on Line 3, youll want to call the db method as an object key/value:

```suggestion
        const newOrder = await Cart.addItemToCart ({ orderId, quantity });
```